src/main/java/edu/kimjones/advancedjava/stock/StockQuoteApplication.java

```diff
@@ -0,0 +1,74 @@
+package edu.kimjones.advancedjava.stock;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Main entry point for the stock quote application.
+ *
+ * @author Kim Jones
+ */
+public class StockQuoteApplication {
+
+    /**
+     * @param args      3 string args are required: one containing a stock symbol followed by two containing dates
+     *                  of the form "mm/dd/yyyy"
+     */
+    public static void main(String[] args) {
+
+        if (args.length != 3) {
+            throw new IllegalArgumentException("Exactly 3 arguments are required!");
+        }
+
```

Comment:
nice input validation. maybe include instructions for proper program execution. 
also consider using Args4j to parse command line. It is a nice little library.